stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/StreamCoilImageLoader.kt

```diff
@@ -0,0 +1,60 @@
+package io.getstream.chat.android.compose.ui.util
+
+import android.content.Context
+import coil.ImageLoader
+import coil.ImageLoaderFactory
+import com.getstream.sdk.chat.coil.StreamImageLoaderFactory
+
+/**
+ * Provides a custom image loader that uses the [StreamImageLoaderFactory] to build the default loading settings.
+ *
+ * Gives support to load GIFs.
+ * */
+internal object StreamCoilImageLoader {
```

Comment:
Let's review this later, but if it works for now, we can ship it (`internal` anyway)

Reply:
Yeah my goal would be to have the already `internal` API as public/reusable element, so that I don't have to C/P the solution here.